client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -83,36 +83,50 @@ const VerticalSelect: FunctionComponent< StepProps > = ( {
 		}
 	};
 
+	const loadingMessage = [
+		{
+			label: '',
+			category: NO__( 'Loading, please wait…' ),
+		},
+	];
+
+	let suggestions: Suggestion[];
+
+	if ( ! normalizedInputValue ) {
+		suggestions = verticals
+			.filter( vertical => popular.includes( vertical.label ) )
+			.map( vertical => ( { ...vertical, category: NO__( 'Popular' ) } ) );
+		resetSiteVertical();
```

Comment:
Moved the reset here instead of having it at `onBlur` to fix a preexisting bug: user could have cleared the input field and the _Next_ button was still active but when pressed nothing happened except it became disabled.